app/controllers/user/shares_controller.rb

```diff
@@ -23,15 +23,15 @@ def new
                Share.new
              end
 
-    @organisations = OrganisationUser.all
+    @organisations = Organisation.all
   end
 
   # POST /shares
   def create
```

Comment:
should we check that they're allowed to do this?

Reply:
Probably - but maybe later